osquery/CMakeLists.txt

```diff
@@ -53,12 +65,14 @@ elseif(FREEBSD)
 endif()
 
 # Add default linking details (the first argument means SDK + core).
-ADD_OSQUERY_LINK_CORE("-rdynamic")
-foreach(DISTRO ${OSQUERY_REQUIRE_RUNTIMES})
-  if(${OSQUERY_BUILD_DISTRO} STREQUAL ${DISTRO})
-    ADD_OSQUERY_LINK_ADDITIONAL("-static-libstdc++")
-  endif()
-endforeach()
+if(NOT WIN32)
+  ADD_OSQUERY_LINK_CORE("-rdynamic")
+  foreach(DISTRO ${OSQUERY_REQUIRE_RUNTIMES})
```

Comment:
I don't think this `foreach` needs to be contained within the platform conditional.